### ISSUE TITLE
Add Icons to token library (Anchor, Run, Running, Pause, Stop)

### ISSUE THREAD

**Issue by NIbokeefe** [ni/nimble#1789]
Please add and update the following icons to the token library.

Add:

anchor-link_16x16
tags: square, arrow, open, box

![Image](https://github.com/ni/nimble/assets/106594349/55c1fb5b-f501-4488-8e1b-7470ddc7769b)

run-triangle_16x16
tags: arrow, play, go

![Image](https://github.com/ni/nimble/assets/106594349/fa3472a9-c7b8-4690-849f-d4286407eecb)

pause_16x16
tags: stop, hold 

![Image](https://github.com/ni/nimble/assets/106594349/2e410023-a6a5-43d1-aff1-bf617b8b782a)

stop-square_16x16
tags: quit, end

![Image](https://github.com/ni/nimble/assets/106594349/b8185ea3-cc88-4efe-981c-1fd2a6e0a423)

UPDATE:

running-arrow_16x16
tags: play, continue, run

![Image](https://github.com/ni/nimble/assets/106594349/1c756abe-92ab-4492-9ea9-aa417482e973)

**Comment by jattasNI** [comment]
@NIbokeefe I noticed some of the icons you attached are color. Our guidance so far has been to build all of our icons in black and then allow clients to apply any of the severity colors. We advise clients not to pick a different color, but offer a way for them to do so at their own risk.

Do you think we could apply our existing guidance/pattern to these icons or do we need new guidance? It looks like the red and green that you picked are the same as the "success" and "error" severity. Should we advise clients to always set the stop icon to "severity=error"? Leave it black and let them decide? Or build new infrastructure so that the source image is always red?

**Comment by NIbokeefe** [comment]
My guidance has always been that if color was applied by design it was intentional. For those colored icons we've always kept them their default color on the UX side.
<img width="1320" alt="Screen Shot 2024-01-31 at 8 48 48 AM" src="https://github.com/ni/nimble/assets/106594349/7835c2db-0f32-4bbe-ad9a-7f89b1271a45">

Early on I asked about why the icons were being converted to black and from my understanding there was a technical reason at that time.  I didn't really push back but I have noticed from when I've seen implemented them they never get colored as I intended. Especially for connected/disconnected series of icons and ones that are half colored which were very intentional:

<img width="70" alt="Screen Shot 2024-01-31 at 9 01 41 AM" src="https://github.com/ni/nimble/assets/106594349/719ecfe1-5292-4bd3-adbb-a7ea94275770">
<img width="66" alt="Screen Shot 2024-01-31 at 9 01 48 AM" src="https://github.com/ni/nimble/assets/106594349/77872b6d-b324-4ad6-8b06-bb28cb2ecae3">

Red can mean: Fail, Error, Alert, and is legacy for how we've drawn stop.  That button should always be easy to find.
Success Green is unique to our variety of brand greens which can mean:  Success, Good, Working, etc.  Run is also one of those icons that by legacy are always meant to be green and sorta primary in that it's easy to distinguish and locate (run is unique in that there are multiple states of running in our applications).

With contrast-accessibility restraints which are difficult to achieve correctly, it doesn't make sense to have varieties of these colors, so we use one version red, green, orange, or purple for a variety of situations.  Whether it's a true "status" or just an alert.

I don't believe we should leave it up to anyone to decide the color for anything that's intended to be colored by default, but I do feel it's okay to allow the flexibility that someone can recolor an icon in their own context when it makes sense for their situation.
